frontend/views/pages/GroupSettings.vue

```diff
@@ -59,7 +59,9 @@ page(pageTestName='dashboard' pageTestHeaderName='groupName' v-if='currentGroupS
       ) Save changes
 
   page-section(title='Leave Group')
-    i18n(tag='p') This means you will stop having access to the #[b group chat] (including direct messages to other group members) and #[b contributions]. Re-joining the group is possible, but requires other members to vote and reach an agreement.
+    p
+      | {{ L('This means you will stop having access to the') }} #[b {{ L('group chat') }}] {{ L('including direct messages to other group members) and') }} #[b {{ L('contributions') }}].
```

Comment:
No using `#[b` syntax with `L` functions in Pug. The only reason to use `L` functions like this in Pug is if the `#[b` syntax is broken. So use `<b>` etc as mentioned in the comment in the other PR

Reply:
Also, splitting up an English string like this will break the translations, since it will result in several strings and sentence fragments being created instead of a single string to be translated.

So there needs to be only a single use of the `L` function here (and again, using the `L` function in Pug is only needed when you're passing in string literals as props to a component, or like here we need to add styling, since `#[b` syntax is broken, otherwise using `i18n` would be more appropriate).

Reply:
Hum not sure if I understood but changed.

Reply:
I don't see the changes. It's still using `L` multiple times and still using `#[b`

Reply:
fixed finally!